packages/db-kit/src/utils/fetchExternal.ts

```diff
@@ -57,118 +52,25 @@ export async function fetchExternal({
   });
 }
 
-/**
- * Accepts a TruffleConfig and returns a priority-sorted list of fetchers based
- * on provided configuration and whether available fetchers support the current
- * blockchain network.
- * @dev HACK this currently only supports EtherscanFetcher
- */
-export async function forTruffleConfig(options: {
-  config: TruffleConfig;
-}): Promise<Fetcher[]> {
-  const { config } = options;
-  const networkId = parseInt(config.network_id);
-
-  // respect optional `config.sourceFetchers`, used to specify explicit
-  // ordering for which fetchers to use.
-  //
-  // since this HACK-ily only supports Etherscan, this logic serves to respect
-  // configuration that explicitly excludes use of Etherscan.
-  const constructors: FetcherConstructor[] = config.sourceFetchers
-    ? config.sourceFetchers.map(sourceFetcherName => {
-        const Fetcher = Fetchers.find(
-          Fetcher => Fetcher.fetcherName === sourceFetcherName
-        );
-
-        if (!Fetcher) {
-          throw new Error(
-            `Unknown external source servce ${sourceFetcherName}.`
-          );
-        }
-        return Fetcher;
-      })
-    : Fetchers;
-
-  // construct fetchers for networkId
-  const fetchers = await Promise.all(
-    constructors
-      // HACK only support EtherscanFetcher right now
-      .filter(({ fetcherName }) => fetcherName === "etherscan")
-      .map(async (Fetcher: FetcherConstructor) => {
-        const options = config[Fetcher.fetcherName];
-
-        // don't fail on unsupported networks; that's expected behavior
-        try {
-          return await Fetcher.forNetworkId(networkId, options);
-        } catch (error) {
-          if (!(error instanceof InvalidNetworkError)) {
-            throw error;
-          }
-        }
-      })
-  );
-
-  return fetchers.filter((fetcher): fetcher is Fetcher => !!fetcher);
-}
-
 async function fetchAndCompile(options: {
```

Comment:
This can probably now be inlined into `fetchExternal` above

Reply:
I mean this code is deleted because there's no need for it, `fetch-and-compile` already handles this.  Or am I misunderstanding?

Reply:
well, my reasoning is that I want the entrypoint to @truffle/fetch-and-compile to be called `fetchAndCompile`. So I don't want another function here that is also called `fetchAndCompile`. So I started looking at it, and it seems like it's been sufficiently gutted that we can just get rid of it as a separate layer here.

Reply:
Oh, you mean `fetchAndCompile`!  Sorry, I thought you were talking about the deleted code.

Reply:
OK, did this.